src/basics.ml

```diff
@@ -112,6 +112,13 @@ let is_symb : sym -> term -> bool = fun s t ->
   | Symb(r,_) -> r == s
   | _         -> false
 
+(** [get_symb t] returns [Some s] if [t] is of the form [Symb (s , _)].
+    Otherwise, it returns [None]. *)
+let get_symb : term -> sym option = fun t ->
```

Comment:
Rename into get_sym has the type is sym.

Reply:
Ignore my previous comment: there are other functions using symb instead of sym already.